test/models/deploy_group_test.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../test_helper'
+
+describe DeployGroup do
+  let(:prod_env) { environments(:production_env) }
+
+  it 'should be saved with valid name/env' do
```

Comment:
you can make it simpler to understand by grouping them to their method
`describe "#new" do` / `describe "#create" do`